diff -urN '-x*.orig' midipix-ntapi.orig/src/string/ntapi_tt_dec_utf16_to_uintptr.c midipix-ntapi/src/string/ntapi_tt_dec_utf16_to_uintptr.c
--- midipix-ntapi.orig/src/string/ntapi_tt_dec_utf16_to_uintptr.c	2019-05-03 10:07:59.616653400 +0800
+++ midipix-ntapi/src/string/ntapi_tt_dec_utf16_to_uintptr.c	2019-05-03 15:34:40.651967000 +0800
@@ -110,7 +110,7 @@
 	__in	const wchar16_t *	wch_cap,
 	__out	uintptr_t *		key)
 {
-	return __tt_dec_utf16_to_uint32(
+	return __tt_dec_utf16_to_uint64(
 		wch_arg,wch_cap,key);
 }
 #endif
