diff -urN -x '*.orig' midipix-pemagine.orig/project/extras.mk midipix-pemagine/project/extras.mk
--- midipix-pemagine.orig/project/extras.mk	2019-05-04 08:45:24.435054900 +0800
+++ midipix-pemagine/project/extras.mk	2019-05-04 09:47:11.110690700 +0800
@@ -6,7 +6,6 @@
 CFLAGS_CONFIG 		+= -UWIN64 -U_WIN64 -U__WIN64 -U__WIN64__
 
 LDFLAGS_SHARED 		+= -Wl,--exclude-all-symbols
-LDFLAGS_SHARED 		+= -nostdlib
 
 ifeq ($(HOST_BITS),32)
 	LDFLAGS_SHARED 	+= -Wl,--entry -Wl,$(HOST_UNDERSCORE)pe_entry_point@12
